apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java

```diff
@@ -908,4 +957,115 @@ public void run() {
          throw new RuntimeException("After " + retryCountLimit + " retries: " + lastException);
       }
    }
+   
+   /**
+    * Gets manifest Information for DLO uploaded file
+    * @param container
+    * @param objectName
+    * @return manifest file name if DLO uploaded, else null
+    */
+   private String getManifestInfo(String container, String objectName) {
+      ObjectApi objectApi = api.getObjectApi(regionId, container);
+      Multimap<String, String> objInfo = objectApi.getWithoutBody(objectName).getHeaders();
+      if (objInfo.containsKey("X-Object-Manifest")) {
+         Collection<String> objectManifest = objInfo.get("X-Object-Manifest");
+         return String.valueOf(objectManifest.toArray()[0]);
+      }
+      return null;
+   }
+   
+   /**
+    * Gets X-Static-Large-Object for SLO uploaded file
+    * @param container
+    * @param objectName
+    * @return true if object is SLO uploaded else null
+    */
+   private String getSLOInfo(String container, String objectName) {
+      ObjectApi objectApi = api.getObjectApi(regionId, container);
+      Multimap<String, String> objInfo = objectApi.getWithoutBody(objectName).getHeaders();
+      if (objInfo.containsKey("X-Static-Large-Object")) {
+         Collection<String> objectManifest = objInfo.get("X-Static-Large-Object");
+         return String.valueOf(objectManifest.toArray()[0]);
+      }
+      return null;
+   }
+   
+   
+   /**
+    * @param containerAndPrefix
+    */
+   private void removeObjectsWithPrefix(String containerAndPrefix) {
+      String[] parts = splitContainerAndKey(containerAndPrefix);
+
+      String container = parts[0];
+      String prefix = parts[1];
+
+      removeObjectsWithPrefix(container, prefix);
+   }
+
+    
+   /**
+    * @param containerAndKey
+    * @return array of size 2 with container and objectPrefix
+    */
+   private String[] splitContainerAndKey(String containerAndKey) {
+      String[] parts = containerAndKey.split("/", 2);
+      checkArgument(parts.length == 2, "No / separator found in \"%s\"", containerAndKey);
+      return parts;
+   }
+    
+   /**
+    * Retrieves a list of existing storage
+    * @param container
+    * @param options
+    * @return maximum of 10,000 object names per request.
+    */
+   private PageSet<? extends StorageMetadata> list(final String container,
```

Comment:
Deleting DLO objects requires, the details of the segments.

BlobStore's List command takes BlobStore's ListContainerOptions as input.
But incase of remove blob we need Swift's ListContainerOptions. That the reasonf i copied and made few changes